Documentation CYF 1.0/pages/api-functions-main.html

```diff
@@ -235,6 +235,15 @@ <h4>All-script functions</h4>
 </pre>
             </div>
 
+			<p>
+                <br><span class="function"><span class="new066"></span><span class="CYF"></span> CreateState(<span class="string"></span> stateName) [E/M/W]</span>
+                This function creates a <span class="term">state</span> with the name <span class="term">stateName</span> which you can enter 
```

Comment:
I don't know why you used `stateName` instead of `name` which works fine, but that isn't too important...
This is such a wordy description and is all over the place.
`This function creates a custom state with the name "name", which will function like the NONE state.
You cannot have two states with the same name.`
This is much clearer and more concise.

Reply:
Should I keep the best practices section?

Reply:
Personally I don't think so, it seems like common sense

Reply:
Got it.